pkg/forwarder/forwarder.go

```diff
@@ -553,7 +553,12 @@ func (f *DefaultForwarder) SubmitConnectionChecks(payload Payloads, extra http.H
 func (f *DefaultForwarder) SubmitOrchestratorChecks(payload Payloads, extra http.Header, payloadType int) (chan Response, error) {
 	bumpOrchestratorPayload(payloadType)
 
-	return f.submitProcessLikePayload(endpoints.OrchestratorEndpoint, payload, extra, true)
+	endpoint := endpoints.OrchestratorEndpoint
+	if config.Datadog.IsSet("orchestrator_explorer.use_legacy_endpoint") {
```

Comment:
We should probably use `GetBool` instead of `IsSet`. If we set
```
orchestrator_explorer:
  use_leagey_endpoint: false
```

we're still pass the check.

Reply:
indeed, but in this case this setting would not be exposed / only for internal use so I don't think it would be an issue (we would remove it in the next release)